src/components/ProjectsCard/ProjectsCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from "@components/Button/Button";
+import { Text } from "@components/Text/Text";
+import { IconNames } from "@data/interfaces/Icons";
+import styles from "./ProjectsCard.module.css";
+import { IProjectsCardProps } from "@data/interfaces/Card";
+const ProjectsCard = ({
+  projectTitle,
+  projectDescription,
+  projectUrl,
+  projectInfoUrl,
+  projectPictureUrl,
+  cardOrientation,
+}: IProjectsCardProps) => {
+  const isReversed = cardOrientation === "reversed";
```

Comment:
```tsx
const cardDirection = cardOrientation === "reversed" ? styles.reversed : styles.normal;
```

Reply:
I also want you to do card direction thing as a data attribute instead of a classname